File: frontend/src/app/models/DetalleSolicitudExtendidoResponse.ts
import {DetalleSolicitudResponse} from "./DetalleSolicitudResponse";

export interface DetalleSolicitudExtendidoResponse {

  idSolicitud: number;
  cite: string;
  fecha: string;
  descripcion: string;
  detalleSolicitudResponses: DetalleSolicitudResponse[];
}

File: prototipo/src/main/java/com/prototipo/infrastructure/rest/response/DetalleSolicitudExtendidoResponse.java
package com.prototipo.infrastructure.rest.response;

import lombok.*;

import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class DetalleSolicitudExtendidoResponse {
    private Long idSolicitud;
    private String cite;
    private String fecha;
    private String descripcion;
    private String nombreServicio;
    private Double precioTotal;

    private List<DetalleSolicitudResponse> detalleSolicitudResponses;
}
